pkg/middleware/logging/logging_test.go

```diff
@@ -54,31 +53,21 @@ func TestLoggingMiddleware(t *testing.T) {
 	require.Equal(t, "Handling /hello/world", log.logs[0])
 	require.Regexp(t, regexp.MustCompile(`Rendered 200 in \d+ms for /hello/world`), log.logs[1])
 
-	// Proxying request
-	r = httptest.NewRequest("GET", "/fake", nil)
-	w = httptest.NewRecorder()
-	viewProxyServer.CreateHandler().ServeHTTP(w, r)
-	resp = w.Result()
-	require.Equal(t, 404, resp.StatusCode)
-
-	require.Equal(t, "Proxying /fake", log.logs[2])
-	require.Regexp(t, regexp.MustCompile(`Proxied 404 in \d+ms for /fake`), log.logs[3])
-
 	// Proxying disabled
-	viewProxyServer.PassThrough = false
 	r = httptest.NewRequest("GET", "/fake", nil)
 	w = httptest.NewRecorder()
 	viewProxyServer.CreateHandler().ServeHTTP(w, r)
 	resp = w.Result()
 	require.Equal(t, 404, resp.StatusCode)
 
-	require.Equal(t, "Proxying is disabled and no route matches /fake", log.logs[4])
+	require.Equal(t, "Proxying is disabled and no route matches /fake", log.logs[2])
+
+	viewProxyServer = viewproxy.NewServer(targetServer.URL, viewproxy.WithPassThrough(targetServer.URL))
```

Comment:
Is this server needed? I didn't see any asserts below this line.

Reply:
Ah nope, I was trying to see if there was a quick way to get logging working again here but ended up deleting that code. This can be deleted with it. 👍